### PR TITLE
feat: custom login style and privacy policy

### PR DESCRIPTION
**Pull request by gunzip**
added custom login for developer portal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,3 +151,21 @@ to create a 'Service' and a 'Profile' tied to the developer portal accounts.
 
 You should receive an API-Key (in the developer portal) and an email
 with a fake fiscal code you can use to start testing the Digital Citizenship API.
+
+## ADB2C Sign-in / Sign-up form style
```
Comment (cloudify) [inline README.md]: I don't understand this `ADB2C` acronym, can we use something more familiar? what does it mean?

File: README.md [hunk 2]
```diff
@@ -151,3 +151,21 @@ to create a 'Service' and a 'Profile' tied to the developer portal accounts.
 
 You should receive an API-Key (in the developer portal) and an email
```
Comment (cloudify) [inline README.md]: `API-Key` -> `API key`

File: README.md [hunk 3]
```diff
@@ -151,3 +151,21 @@ to create a 'Service' and a 'Profile' tied to the developer portal accounts.
...
 with a fake fiscal code you can use to start testing the Digital Citizenship API.
```
Comment (cloudify) [inline README.md]: question, are the fake fiscal codes actually valid fiscal codes? (i.e. do we have a non-zero probability of some real person having that fiscal code?)

File: web/privacy.html [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+</head>
+
+<body>
+  <div class="container">
+    <div class="privacyPolicy">
+      <h1>Privacy Policy</h1>
+      <div>
+        <p>
+          In questa pagina si descrivono le modalità e le condizioni del trattamento dei dati personali degli utenti che consultano
+          e utilizzano il sito https://agid-apim-prod.portal.azure-api.net (di seguito anche il Sito).
```
Comment (cloudify) [inline web/privacy.html]: replace with `developer.cd.italia.it`

File: web/privacy.html [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+        </p>
+        <p>
+          L’informativa è resa ai sensi dell’art. 13 del D. Lgs. n. 196/2003 - Codice in materia di protezione dei dati personali a
```
Comment (cloudify) [inline web/privacy.html]: add link to the article page

File: web/privacy.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+          tutti gli Utenti che, interagendo con il Sito, forniscono al gestore dello stesso i propri dati personali. La validità
+          dell’informativa contenuta nella presente pagina è limitata al solo Sito e non si estende ad altri siti web eventualmente
+          consultabili mediante collegamento ipertestuale. Il Titolare del trattamento per i dati raccolti attraverso il
+          Sito è AgID - Agenzia per l’Italia Digitale, con sede in via Liszt 21 - 00144 Roma, indirizzo email info@agid.gov.it.
```
Comment (cloudify) [inline web/privacy.html]: link email `mailto:info@agid.gov.it`

File: web/privacy.html [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+          Eventuali soggetti terzi ai quali i dati potrebbero essere comunicati per l’esecuzione di operazioni di trattamento
+          connesse con le finalità specificate nella presente informativa, saranno preventivamente designati dal Titolare
+          quali Responsabili del trattamento ai sensi dell’art. 29 del Codice in materia di protezione dei dati personali.
+          Tutti i dati personali forniti attraverso il Sito saranno trattati in modo lecito e secondo correttezza al fine
+          di fornire i servizi richiesti nonché di rispondere alle comunicazioni ed alle domande degli Utenti, sempre nel
+          perseguimento degli scopi istituzionali dell’Agenzia così come previsti dalla Legge. I dati personali sono trattati
+          con strumenti automatizzati per il tempo strettamente necessario a conseguire gli scopi per cui sono stati raccolti
+          secondo quanto previsto dall’art. 11, lett. e) del Codice privacy. v
```
Comment (cloudify) [inline web/privacy.html]: extra `v` at the end of line

File: README.md (hunk 3 above)
Comment (gunzip) [reply README.md]: fake fiscal codes cannot collide with real ones as we use an "invalid" character ("Y") for the local code: https://github.com/gunzip/digital-citizenship-onboarding/blob/master/src/fake_profile.ts#L17

File: README.md (hunk 1 above)
Comment (gunzip) [reply README.md]: it's 'active directory business to clients (there's a link to the azure docs at the beginning of this readme): https://azure.microsoft.com/it-it/services/active-directory-b2c/'

that's the MS way to call "an active directory where you can login non-microsoft accounts" (ie. with any email)

**Pull request merged**